fv/containers/containers.go

```diff
@@ -44,6 +44,15 @@ type Container struct {
 
 var containerIdx = 0
 
+var etcdImage string
```

Comment:
What about getting rid of the global `etcdImage` variable, renaming `init()` to `getEtcdImage()` and invoking just-in-time?